src/main/java/seedu/address/logic/commands/MarkSupplierCommand.java

```diff
@@ -55,7 +59,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setSupplier(supplierToMark, markedSupplier);
         model.updateFilteredSupplierList(Model.PREDICATE_SHOW_ALL_SUPPLIERS);
-        return new CommandResult(String.format(MESSAGE_MARK_SUPPLIER_SUCCESS, targetIndex.getOneBased(), status));
+        return new CommandResult(String.format(MESSAGE_MARK_SUPPLIER_SUCCESS, Messages.format(supplierToMark), status));
```

Comment:
Should this be formatWithoutStatus instead?

Reply:
Yup!